app/src/main/java/com/celzero/bravedns/service/ConnectionMonitor.kt

```diff
@@ -164,7 +161,7 @@ class ConnectionMonitor(context: Context, networkListener: NetworkListener) :
 
     data class UnderlyingNetworks(val allNet: Set<Network>?, val ipv4Net: Set<Network>,
                                   val ipv6Net: Set<Network>, val useActive: Boolean,
-                                  var isActiveNetworkMetered: Boolean)
+                                                      var `isActiveNetworkMetered`: Boolean, var lastUpdated: Long)
```

Comment:
What do backticks \` mean here for `isActiveNetworkMetered`?

Reply:
`isActiveNetworkMetered` is keyword used in ConnectivityManager, backticks will make sure it's local variable.